ui/fuse.js

```diff
@@ -46,6 +46,7 @@ Sparky.task('config', _ => {
 Sparky.task('clean', _ => Sparky.src('dist/').clean('dist/'));
 Sparky.task('env', _ => (isProduction = true));
 Sparky.task('copy-assets', () => Sparky.src('assets/*.svg').dest('dist/'));
+Sparky.task('copy-assets', () => Sparky.src('assets/*.png').dest('dist/'));
```

Comment:
The same sparky task name? Not sure if it works but it seems risky. I think it can do wildcards, I'd look that up.

Reply:
used a wildcard.